ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -343,3 +382,267 @@ Set RHODS Users Group To Inexistent Group
+    [Arguments]   ${name}
+    Search for accelerator profile in grid    ${name}
+    Wait Until Page Contains Element    ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//*[text()='${name}']/ancestor::tr//button[@aria-label='Kebab toggle']
+    Click Button    ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//*[text()='${name}']/ancestor::tr//button[@aria-label='Kebab toggle']
+    Wait Until Page Contains Element    ${EDIT_ACCELERATOR_PROFILE_BTN}
+    Click Button    ${EDIT_ACCELERATOR_PROFILE_BTN}
+
+
+Click On Enable Accelerator Profile
+    [Documentation]    Click on the the Enable switch from an specific Accelerator Profile
+    ...                view: Settings -> Accelerator profiles
+    [Arguments]   ${name}
+    Wait Until Page Contains Element    //input[@id='${name}-enable-switch']/following-sibling::span
+    Click Element    //input[@id='${name}-enable-switch']/following-sibling::span
+
+Disable Accelerator Profile
+    [Documentation]    Disable an specific Accelerator Profile from the Grid
+    ...                view: Settings -> Accelerator profiles
+    [Arguments]   ${name}
+    Click On Enable Accelerator Profile    ${name}
+    Wait Until Page Contains Element    ${ACCELERATOR_DISABLE_MODAL_DISABLE_BUTTON}
+    Click Button    ${ACCELERATOR_DISABLE_MODAL_DISABLE_BUTTON}
+
+Search for accelerator profile in grid
+    [Documentation]    Search for and specific Accelerator Profile in the grid
+    ...                view: Settings -> Accelerator profiles
+    [Arguments]   ${name}
+    Wait Until Page Contains Element    ${ACCELERATOR_PROFILE_SEARCH_INPUT}
+    Click Element    ${ACCELERATOR_PROFILE_SEARCH_INPUT}
+    Clear element and input text    ${ACCELERATOR_PROFILE_SEARCH_INPUT}   ${name}
+
+Delete Accelerator Profile
+    [Documentation]    Delete an specific Accelerator Profile from the Grid
+    ...                view: Settings -> Accelerator profiles
+    [Arguments]   ${name}
+    Search for accelerator profile in grid    ${name}
+    Wait Until Page Contains Element    ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//*[text()='${name}']/ancestor::tr//button[@aria-label='Kebab toggle']
+    Click Button    ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//*[text()='${name}']/ancestor::tr//button[@aria-label='Kebab toggle']
+    Wait Until Page Contains Element    ${DELETE_ACCELERATOR_PROFILE_BTN}
+    Click Button    ${DELETE_ACCELERATOR_PROFILE_BTN}
+    Wait Until Page Contains Element    ${ACCELERATOR_DELETE_MODAL_INPUT}
+    Input Text    ${ACCELERATOR_DELETE_MODAL_INPUT}    ${name}
+    Click Button    ${ACCELERATOR_DELETE_MODAL_DELETE_BUTTON}
+
+Create An Accelerator Profile Via CLI
+    [Documentation]    Create an instance of Accelerator Profile using OC
+    [Arguments]   ${name}  ${ns}=redhat-ods-applications
+    ${rc}    ${out}    Run And Return Rc And Output   oc create -f ${ACCELERATOR_RESOURCES_DIRPATH}/${name}.yaml -n ${ns}
+    Should Be Equal As Integers    ${rc}     0
+
+Create An Accelerator Profile Via UI
```

Comment:
based on the name of the keyword, I'd assume it clicks the button for creation as first step